deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -0,0 +1,388 @@
+  ImageServerUrl:
+    Type: 'AWS::SSM::Parameter::Value<String>'
+    Description: The url of the IIIF Image Server
+    Default: '/all/stacks/mellon-image-service/url'
+  InfrastructureStackName:
+    Type: String
+    Default: "mellon-app-infrastructure"
+    Description: The name of the parent infrastructure/networking stack that you created. Necessary
+                 to locate and reference resources created by that stack.
+
+Outputs:
+  StateMachine:
+    Value: !Ref StateMachine
+
+  ProcessBucket:
+    Value: !Ref ProcessBucket
+
+  ManifestBucket:
+    Value: !Ref ManifestBucket
+
+Resources:
+
+  CanvasDefaultWidth:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/canvas-default-width"
+      Value: "2000"
+      Description: Canvas default width
+
+  CanvasDefaultHeight:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/canvas-default-height"
+      Value: "2000"
+      Description: Canvas default height
+
+  ImageServerBaseUrl:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/image-server-base-url"
+      Value: "https://STUB:8182/iiif/2"
+      Description: Image server base url
+
+  MainCsv:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/main-csv"
+      Value: "main.csv"
+      Description: CSV name
+
+  SequenceCsv:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/sequence-csv"
+      Value: "sequence.csv"
+      Description: CSV name
+
+  ManifestServerBaseUrl:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/manifest-server-base-url"
+      Value: "https://"
+      Description: CSV name
+
+  ProcessBucketReadBasepath:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/process-bucket-read-basepath"
+      Value: "process"
+      Description: S3 bucket read base path
+
+  ProcessBucketWriteBasepath:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/process-bucket-write-basepath"
+      Value: "finished"
+      Description: S3 bucket write base path
+
+  ImageServerBucketBasepath:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/image-server-bucket-basepath"
+      Value: "images"
+      Description: S3 bucket images base path
+
+  ManifestServerBucketBasepath:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: String
+      Name: !Sub "/all/stacks/${AWS::StackName}/config/manifest-server-bucket-basepath"
+      Value: "manifest"
```

Comment:
Are we sure we want this? Is it necessary? From what I saw, this will create a URL pattern that looks like "https://<base-domain>/manifest/<project-name>/manifest/index.json" - is that what we want?